android/app/src/main/python/mmrelay/log_utils.py

```
@@ -0,0 +1,242 @@
+        try:
+            # Set up size-based log rotation
+            max_bytes = DEFAULT_LOG_SIZE_MB * LOG_SIZE_BYTES_MULTIPLIER
+            backup_count = DEFAULT_LOG_BACKUP_COUNT
+
+            if config is not None and "logging" in config:
+                max_bytes = config["logging"].get("max_log_size", max_bytes)
+                backup_count = config["logging"].get("backup_count", backup_count)
+            file_handler = RotatingFileHandler(
+                log_file, maxBytes=max_bytes, backupCount=backup_count, encoding="utf-8"
+            )
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)